lib/app-providers.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { createTelemetryClient, TelemetryProvider } from "@lib/telemetry";
+import { Provider } from "next-auth/client";
+import { QueryClient, QueryClientProvider } from "react-query";
+import { Hydrate } from "react-query/hydration";
+
+export const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      refetchOnWindowFocus: false, // don't fetch when window is focused
```

Comment:
why not? it's a nice feature

Reply:
i like this too. dont see downside of enabling it